specs/clickhouse.spec

```diff
@@ -305,6 +305,9 @@ fi
 ################################################################################
 
 %changelog
+* Tue Mar 17 2019 Gleb Goncharov <andy@essentialkaos.com> - 19.17.5.18-1
+- Separate clickhouse-client package from the clickhouse-server
```

Comment:
@gongled I'm not sure that this comment describes clearly changes which you made. Maybe something like this?

```Removed dependency of clickhouse-client from clickhouse-server```

Reply:
Sure, why not (ce4156c9b756798223258619a24e6e3fcc24656b).